Sprint-1/interpret/percentage-change.js

```diff
@@ -12,11 +12,12 @@ console.log(`The percentage change is ${percentageChange}`);
 // Read the code and then answer the questions below
 
 // a) How many function calls are there in this file? Write down all the lines where a function call is made
-
+//line 4, line 5, line 10 so 2 unique function calls but 3 used in total
```

Comment:
Is `Number(...)` a function call?

Reply:
Hi @cjyuan according to what I've looked at online and I also prompted GPT, I do belive that Number() is a function call. Isn't it that when we call a function we expect a value to be returned? I'm not sure if my logic makes sense

Reply:
Unless only functions called with a dot operator count as function calls?

Reply:
You mentioned 2 unique function calls, 3 use in total. I wasn't sure how many function calls you meant. Yes, Number() is a function call. So how many function calls are there?

Reply:
`priceAfterOneYear.replaceAll(",", "")` and `console.log(...)` are also function calls.